gui/src/main/java/io/bitsquare/gui/util/MakeSound.java

```diff
@@ -0,0 +1,75 @@
+package io.bitsquare.gui.util;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.DataLine;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.SourceDataLine;
+
+public class MakeSound {
+
+    private final int BUFFER_SIZE = 128000;
+    private File soundFile;
+    private AudioInputStream audioStream;
+    private AudioFormat audioFormat;
+    private SourceDataLine sourceLine;
+
+    /**
+     * @param filename the name of the file that is going to be played
+     */
+    public void playSound(String filename){
+
+        String strFilename = filename;
+
+        try {
+            soundFile = new File(strFilename);
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.exit(1);
```

Comment:
instead of a stacktrace and an exit, just log the error. Bitsquare shouldn't crash because a sound is not playing!